lib/client.js

```diff
@@ -101,9 +101,10 @@ extend(Raven.prototype, {
     }
 
     this.onFatalError = this.defaultOnFatalError = function (err, sendErr, eventId) {
-      console.error(err.stack);
+      console.error(err && err.stack ? err.stack : err);
```

Comment:
maybe, but for same "people can throw dumb stuff" reason as before I'd rather just leave it explicit like this